packages/web-components/src/components/card-group/__stories__/card-group.stories.ts

```diff
@@ -139,7 +139,7 @@ const gridModes = {
 export const withCardInCard = ({ parameters }) => {
   const { cards, gridMode } = parameters?.props?.CardGroup ?? {};
   return html`
-    <dds-card-in-card href="https://example.com">
+    <dds-card-in-card grid-mode="${ifNonNull(gridMode)}" href="https://example.com">
```

Comment:
```suggestion
    <dds-card-in-card grid-mode="${gridMode}" href="https://example.com">
```

Just a simple point, we don't actually need this function if we're using select for the knob.